From 3ffcffcb5e482a89dc29b26891158766de44c877 Mon Sep 17 00:00:00 2001
From: Kristoffer Flores <kflores@blackberry.com>
Date: Mon, 16 Sep 2013 10:54:22 -0400
Subject: [PATCH 30/61] Add read_device_identifying_information to
 com.blackberry.plugin

Reviewed by Bryan Higgins <bhiggins@blackberry.com>
Tested by Daniel Audino <daudino@blackberry.com>
---
 plugin/com.blackberry.identity/plugin.xml |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/plugin/com.blackberry.identity/plugin.xml b/plugin/com.blackberry.identity/plugin.xml
index e584dd7..47dfd7e 100644
--- a/plugin/com.blackberry.identity/plugin.xml
+++ b/plugin/com.blackberry.identity/plugin.xml
@@ -24,9 +24,12 @@
     </js-module>
 
     <platform name="blackberry10">
-        <source-file src="src/blackberry10/index.js" />
+        <config-file target="www/config.xml" parent="/widget/rim:permissions">
+            <rim:permit>read_device_identifying_information</rim:permit>
+        </config-file>
         <config-file target="www/config.xml" parent="/widget">
             <feature name="com.blackberry.identity" value="com.blackberry.identity" />
         </config-file>
+        <source-file src="src/blackberry10/index.js" />
     </platform>
 </plugin>
-- 
1.7.10.msysgit.1

